projects/js-packages/licensing/components/activation-screen-controls/style.scss

```
@@ -95,46 +106,30 @@
 	}
 
 	&:hover {
-		background-color: var(--jp-green-60);
-		border-color: var(--jp-green-60);
-		color: var(--jp-white);
+		/* Gray / Gray 80 */
+		background-color: #2C3338;
+		/* Black & White / White */
+		color: #FFFFFF;
 	}
 
 	&:focus {
-		background-color: var(--jp-green-70);
-		border-color: var(--jp-white);
-		box-shadow: 0 0 0 1px var(--jp-green-70);
-		color: var(--jp-white);
-	}
-
-	&:active {
-		background: var(--jp-green-60);
-		border-color: var(--jp-green-60);
-		color: var(--jp-white);
+		/* Gray / Gray 80 */
+		background-color: #2C3338;
+		/* Black & White / White */
+		color: #FFFFFF;
+		border: 1px solid #FFFFFF
 	}
 
 	&[disabled],
 	&:disabled {
-		background: var(--jp-gray);
-		color: #a7aaad;
-		border-color: var(--jp-gray);
+		/* Gray / Gray 5 */
+		background-color: #DCDCDE;
+		/* Gray / Gray 20 */
+		color: #A7AAAD;
```

`#DCDCDE` seems to be `--jp-gray`, so you may not have to change anything here.
https://github.com/Automattic/jetpack/blob/d53bd6a3d3f56fbf04ff04c2192d7dc7de352bc3/projects/js-packages/base-styles/style.scss#L13

```suggestion
		background-color: var(--jp-gray);
		color: var(--jp-gray-20);
```